README.md

```diff
@@ -107,13 +202,11 @@ Custom contract is any type of smart contract that was developed outside of Holo
 1. Install dependencies with `yarn install`.
 1. Initialize the project with `yarn run init` _(this will copy sample environment configs)_.
 
-### Building
-
-All smart contracts source code is located in the `src` directory.
+### Building 
 
-Files from the `src` directory are automatically transpiled into the `contracts` directory each time that **hardhat** compiles the contracts.
+When the project is built, the code in the `src` folder gets written to the `contracts` folder. The files in the `contracts` folder are the "real" files that are used for testing and code verification on all the scanners.
 
-To manually run just the build task use `yarn run build`.
+Again, files from the `src` directory are automatically transpiled into the `contracts` directory each time that **hardhat** compiles the contracts.
 
 ### Compiling, Testing, and Deploying (Locally)
```

Comment:
Can we remove this section. This is not the advisable way to develop locally in the application. I believe it will be confusing for C4 wardens, and imo makes us look silly